docs/en/operations/settings/settings_users.md

```diff
@@ -54,27 +54,27 @@ Password could be specified in plaintext or in SHA256 (hex format).
 
 ### user_name/networks
 
-List of networks which the user can connect to ClickHouse server from.
+List of networks that the user can connect to the ClickHouse server from.
 
-Each element of list has one of the following forms:
+Each element of the list has one of the following forms:
```

Comment:
Looks a bit nicer as `Each element of the list can have one of the following forms`.